src/Http/Controllers/LedgerAccountController.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace FluxErp\Http\Controllers;
+
+use FluxErp\Actions\LedgerAccount\CreateLedgerAccount;
+use FluxErp\Actions\LedgerAccount\DeleteLedgerAccount;
+use FluxErp\Actions\LedgerAccount\UpdateLedgerAccount;
+use FluxErp\Helpers\ResponseHelper;
+use FluxErp\Http\Requests\CreateLedgerAccountRequest;
+use FluxErp\Models\LedgerAccount;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Request;
+use Illuminate\Validation\ValidationException;
+
+class LedgerAccountController extends BaseController
+{
+    public function __construct()
+    {
+        parent::__construct();
+        $this->model = new LedgerAccount();
+    }
+
+    public function create(CreateLedgerAccountRequest $request): JsonResponse
+    {
+        try {
+            $ledgerAccount = CreateLedgerAccount::make($request->all())
+                ->execute();
+        } catch (ValidationException $e) {
+            return $e->getResponse();
+        }
+
+        return ResponseHelper::createResponseFromBase(
+            statusCode: 201,
+            data: $ledgerAccount,
+            statusMessage: 'ledger account created'
+        );
+    }
+
+    public function update(Request $request): JsonResponse
+    {
+        $data = $request->all();
+        if (! array_is_list($data)) {
+            $data = [$data];
+        }
+
+        $responses = [];
+        foreach ($data as $key => $item) {
+            try {
+                $responses[] = ResponseHelper::createArrayResponse(
+                    statusCode: 200,
+                    data: $ledgerAccount = UpdateLedgerAccount::make($item)->validate()->execute(),
+                    additions: ['id' => $ledgerAccount->id]
+                );
+            } catch (ValidationException $e) {
+                $responses[] = ResponseHelper::createArrayResponse(
+                    statusCode: 422,
+                    data: $e->errors(),
+                    additions: [
+                        'id' => array_key_exists('id', $item) ? $item['id'] : null,
+                    ]
+                );
+
+                unset($data[$key]);
+            }
+        }
+
+        return ResponseHelper::createResponseFromArrayResponse($responses);
```

Comment:
this does not work. createResponseFromArrayResponse() expects an array with a status key and either responses or errors or data key